SETUP.md

```diff
@@ -249,9 +245,9 @@ This script does all of the steps described in the *Manual setup* section below.
 
 You must install three packages as libraries from PyPI:
 
-* `azure-cli`
-* `azureml-sdk[databricks]`
-* `pydocumentdb`
+* `azure-cli==2.0.56`
```

Comment:
here there are some versions that are not the same as in the conda installer. I agree that is safer to fix the versions. I think we are going to forget that there are different versions. Can't think of a good solution for this :-(

Reply:
I think trying to keep pydocumentdb and azure-cli on same version as rest of repo is good, as long as we can test it :).

The bigger issue is azureml-sdk[datbricks]; it's not exactly the same as azureml-sdk, so I'm ok with having it as a different version, but at the same time, we need to decide on the resolution of the authentication issues that are popping up in places like #529 so that we can eventually get to a place of being current with it

Reply:
makes sense